server/ansible/roles/pbench-server-enable/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Example: /var/www/html/pbench-results-host-info.versioned/pbench-results-host-info.URL002
+#          ^-----------^ ^-------------------pbench_host_info_prefix-------------------^
+#                ^
+#     httpd_document_root_dir
+- name: "set up links to the active files"
+  file:
+    path: "{{ httpd_document_root_dir }}/{{ pbench_host_info_prefix }}{{ item }}"
+    src:  "{{ pbench_host_info_prefix | basename }}{{ item }}.active"
+    state: link
+    force: yes
+  with_items:
+    - "002"
+
+- name: "enable cron jobs"
```

Comment:
Do we want to let the admin do this when they are ready?

Reply:
Not sure: shouldn't enabling the server be a single step? E.g. the doc would say "run the pbench-server-enable playbook, after the crontab has been checked." Are you proposing a two-step enablement? Or something else?

Reply:
I made the crontab steps to be separate roles but the enable/display playbook still run both roles. It would be easy to just create separate playbooks, but I'm not sure I see the point.